tests/download_noaa_aws.sh

```diff
@@ -25,13 +25,30 @@ if [[ "$8" == "" && -z "$GEOIPS_PACKAGES_DIR" ]]; then
 fi
 
 if [[ "$1" == "" || "$bad_command" == "1" ]]; then
-    echo "Usage: $0 <satellite> YYYY MM DD HH MN <testdata_dir> <rclone_conf>"
-    echo "    satellite: goes16, goes17, himawari8, himawari9, geokompsat"
-    echo "    testdata_dir: if 'default' or not specified, defaults to: "
+    echo "Usage: $0 <satellite> YYYY MM DD HH MN <testdata_dir> <rclone_conf> <wildcard_list>"
+    echo "    satellite:"
+    echo "        goes16"
+    echo "        goes17"
+    echo "        himawari8"
+    echo "        himawari9"
+    echo "        geokompsat"
+    echo "        noaa-20"
+    echo "        noaa-21"
+    echo "        snpp"
+    echo "        jpss"
+    echo "    testdata_dir: "
+    echo "        if 'default' or not specified, defaults to: "
     echo "       \$GEOIPS_TESTDATA_DIR/test_data_noaa_aws/data/<satellite>/<YYYYmmdd>/<HHMN>"
-    echo "    rclone_conf: if 'default' or not specified, defaults to:"
+    echo "    rclone_conf: "
+    echo "        if 'default' or not specified, defaults to:"
     echo "        \$GEOIPS_PACKAGES_DIR/geoips/setup/rclone_setup/rclone.conf"
-    echo "    wildcard_list: list of strings to match in filenames (ie, channels). Defaults to all files for dtg"
+    echo "    collection:"
+    echo "        viirs"
+    echo "           VIIRS-IMG-GEO-TC"
+    echo "           VIIRS-I5-SDR"
```

Comment:
How do we differentiate between "collection' and "wildcard_list" ?  Is collection used for all of the satellites?

"collection' is the subdirectory, and "wildcard_list" are substrings from the filenames?  Is this always true?

Reply:
I'm not entirely sure if "collection" is the right terminology, but yes, that defines the subdirectory under the S3 bucket. "wildcard_list" only applies to the substrings in the filenames. I think that will always be true, but I could be wrong.